extensions/websockets-next/deployment/pom.xml

```diff
@@ -96,7 +96,8 @@
             <properties>
                 <maven.compiler.target>21</maven.compiler.target>
```

Comment:
why are these even set to 21 and not 17?

Reply:
They are a Maven profile for specific tests